README.md

```diff
@@ -1,16 +1,26 @@
 ## 18F Pages
 
-This repo contains tools and instructions to generate
-[Jekyll](http://jekyllrb.com/)-based [18F](https://18f.gsa.gov/) sites
-automatically from [GitHub](https://github.com/) repositories in a similar
-fashion to [GitHub pages](https://pages.github.com/).
+[18F Pages](https://pages.18f.gov/) is the platform that
+[18F](https://18f.gsa.gov/) uses to prototype and publish many of its
+[Jekyll](http://jekyllrb.com/)-based web sites. It works very similarly to
+[GitHub pages](https://pages.github.com/).
+
+This repo contains both the Jekyll source for
+[https://pages.18f.gov/](https://pages.18f.gov/) and the
```

Comment:
Would be cleaner to just let pages URL get [autolinked][1].

[1]: https://help.github.com/articles/github-flavored-markdown/#url-autolinking